resources/definitions/os_detection/ericsson-6600.yaml

```diff
@@ -6,6 +6,7 @@ discovery:
     -
         sysObjectID: 
             - .1.3.6.1.4.1.193.81.1.1.3
+            - .1.3.6.1.4.1.193.81.1.1.1
```

Comment:
Instead of listing devices individually, just remove the minor version oid at the end. So this would be instead:

```
    -
        sysObjectID: 
            - .1.3.6.1.4.1.193.81.1.1.
```